src/main/java/com/xellitix/jenkins/plugintool/cli/converter/PluginListOutputFormatConverter.java

```diff
@@ -0,0 +1,46 @@
+package com.xellitix.jenkins.plugintool.cli.converter;
+
+import com.xellitix.jenkins.plugintool.output.PluginListOutputFormat;
+
+import java.util.Arrays;
+
+/**
+ * {@link PluginListOutputFormat} converter.
+ *
+ * <p>
+ * Used for converting output data to specified format.
```

Comment:
This class is actually used to parse a PluginListOutputFormat specified by the user.